src/main/resources/application.conf

```diff
@@ -455,10 +455,19 @@ mantis {
     filter-manager-query-timeout = 3.minutes
   }
 
-  extvm {
-    host = "127.0.0.1"
-    port = 8888
+  vm {
+    # local | external
+    mode = "local"
```

Comment:
Maybe `s/local/internal/g` as "local" could be interpreted as running on the same machine?